### PR TITLE
feat: supertokens integration

### PR DESCRIPTION
**Pull request by reenphygeorge**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,13 +16,17 @@
     "author": "",
     "license": "ISC",
     "dependencies": {
+        "@fastify/cors": "7.0.0",
         "@fastify/env": "^3.0.0",
+        "@fastify/formbody": "6",
         "@fastify/swagger": "^6.0.0",
         "@prisma/client": "^3.14.0",
+        "ascertain": "0.7.19",
```
Comment (akhilmhdh) [inline package.json:24]: What's this package

Comment (akhilmhdh) [reply package.json:24]: @seramg @reenphygeorge

Comment (seramg) [reply package.json:24]: By mistake... Will resolve ASAP

**Comment by akhilmhdh** [comment]
@reenphygeorge @seramg  rebase and remove the unnecessary module

**Comment by akhilmhdh** [comment]
@seramg rebase the codebase

**Pull request closed without merging**